FreeRTOS/Demo/RISC-V_RV32_SiFive_HiFive1-RevB_FreedomStudio/main.c

```diff
@@ -137,6 +150,16 @@ struct metal_interrupt *pxInterruptController;
 	configASSERT( pxInterruptController );
 	metal_interrupt_init( pxInterruptController );
 
+	#if( mainVECTOR_MODE_DIRECT == 1 )
```

Comment:
Does this hardware work with both direct and vectored configurations - or can we just hard code to whichever it uses?

Reply:
Yes, it works with both modes.